src/client/handlers/AuditEventsMultiPathGet.ts

```diff
@@ -0,0 +1,117 @@
+import type { ContextTimed } from '@matrixai/contexts';
+import type { ClientRPCRequestParams, ClientRPCResponseResult } from '../types';
+import type {
+  AuditEvent,
+  AuditEventSerialized,
+  AuditEventToAuditEventSerialized,
+  TopicPath,
+} from '../../audit/types';
+import type { Audit } from '../../audit';
+import type { AuditEventId, AuditEventIdEncoded } from '../../ids';
+import { ServerHandler } from '@matrixai/rpc';
+import * as auditUtils from '../../audit/utils';
+
+class AuditEventsGet extends ServerHandler<
+  {
+    audit: Audit;
+  },
+  ClientRPCRequestParams<{
+    paths: Array<string>;
+    seek?: AuditEventIdEncoded | number;
```

Comment:
Right now the type for `paths` is `Array<string>`, but it might make more sense to have it as `Array<TopicPath>`.